docs/pages/components/Panel.mdx

```diff
@@ -0,0 +1,261 @@
+---
+title: Panel
+description: A common, versatile pattern used to display content composed of different elements, usually in a collection.
+source: https://github.com/dequelabs/cauldron/blob/develop/packages/react/src/components/Panel/index.tsx
+---
+
+import {
+  Panel,
+  PanelHeader,
+  PanelContent,
+  Toast,
+  Button,
+  Icon,
+  Link,
+  IconButton,
+  Code
+} from '@deque/cauldron-react';
+
+```js
+import { Panel, PanelHeader, PanelContent } from '@deque/cauldron-react';
+```
+
+## Examples
+
+### Panel
+
+At its simplest, a `Panel` is a styled semantic HTML `<section>` element with with content inside.
+
+<Note>
+If a `Panel` does not have a heading, <Link href="https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/general-principles.html" rel="noopener noreferrer" target="_blank">it is recommended</Link> to add an `aria-label` to the `Panel` component.
+</Note>
```

Comment:
I'm not sure this section is needed or may need to be reworded. Overused landmarks [can be problematic](https://webaim.org/blog/aria-cause-solution/) so we want to tread lightly in how strongly we want to encourage it. It might be better to say, "If your section needs to have an accessible name, be sure to include an `aria-label` or `aria-labelledby` pointed to the id of your label."

Reply:
Not sure why this change didn't get committed - unresolving and fixing now.